### ISSUE TITLE
Visitors have chains

### ISSUE THREAD

**Issue by GlitchedTurtle** [Aviuz/PrisonLabor#69]
Visitors have the prisoner chains debuff

The chains also reduce the value of the prisoners

**Comment by Aviuz** [comment]
https://github.com/OrionFive/Hospitality/issues/104

**Comment by GlitchedTurtle** [comment]
"The chains also reduce the value of the prisoners"

**Comment by Aviuz** [comment]
@GlitchedTurtle 
I understand that, but all is described in linked Issue.

If that's not enough, let me explain:
* Whole system that I've implemented is focused on changing deep-core movement slow (that is even not visible sometimes, that's why this bug persists so long in Hospitality mod) to Hediff (it's like debuff) which **also** reduce manipulation
* Manipulation reduction appear **only and always** with movement reduction.
* Whole bug is reasoned by weird implementation of restraints system in RimWorld base game.
* All pawns are restraint **if** InRestraints condtions is met
``C#
// RimWorld.RestraintsUtility
public static bool InRestraints(Pawn pawn)
{
	if (!pawn.Spawned)
	{
		return false;
	}
	if (pawn.HostFaction == null)
	{
		return false;
	}
	Lord lord = pawn.GetLord();
	return lord == null || lord.LordJob == null || !lord.LordJob.NeverInRestraints;
}
``
* As you can see author missed one thing that actually visitors meet that condition. It's worth mentioning that modders do not study all source code and it's fine. Especially when visitors hadn't got the "In restraints" label in info because weird function:
``C#
// RimWorld.RestraintsUtility
public static bool ShouldShowRestraintsInfo(Pawn pawn)
{
	return pawn.IsPrisonerOfColony && RestraintsUtility.InRestraints(pawn);
}
``


To sum up: bug is completely on Hospitality side. It's understandable that that was very hard to find out and it's looking confusing. Author is aware and will fix this up.

**Comment by GlitchedTurtle** [comment]
ok sorry, didn't think you saw the second point

**Comment by OrionFive** [comment]
It's fixed for the upcoming B18 version of Hospitality. Thanks for reporting it.
